contributors/contributors/github-sponsorship.md

```diff
@@ -0,0 +1,12 @@
+# Mattermost Github sponsorship
+At Mattermost, we would like to empower our key contributors and sponsor them for 1 year. This is not a financial reward, but mostly a recognition.
+With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors)  allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
+## Who is eligible?
+_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by Github.
+## Process outline:
+1. MM staff member recommends someone for sponsorship and writes 1-2 sentences about why.
+2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec approves
+3. Request sent to finance team
+4. Email sent to potential sponsor telling them what to do.
```

Comment:
```suggestion
4. An email is sent to the potential sponsor letting them know about the next steps in the process.
```

Reply:
If "sponsor" is the person being sponsored then I recommend rewording to:
"An email is sent to the contributor letting them know about the sponsorship and the next steps in the process."